aspnetcore/publishing/iis.md

```diff
@@ -419,15 +430,15 @@ Troubleshooting:
 
 * Confirm that the application runs locally on Kestrel. A process failure might be the result of a problem within the application. For more information, see [Troubleshooting tips](#troubleshooting-tips).
 
-* Check the *processPath* attribute on the `<aspNetCore>` element in *web.config* to confirm that it is *dotnet* for a framework-dependent deployment or *.\my_application.exe* for a self-contained deployment.
+* Check the *processPath* attribute on the `<aspNetCore>` element in *web.config* to confirm that it's *dotnet* for a framework-dependent deployment or *.\my_application.exe* for a self-contained deployment.
 
 * For a framework-dependent deployment, *dotnet.exe* might not be accessible via the PATH settings. Confirm that *C:\Program Files\dotnet\* exists in the System PATH settings.
```

Comment:
Use "framework-dependent deployment (FDD)" and "self-contained deployment (SCD)" on the 1st occurrence of each to avoid typing it out each time.